### ISSUE TITLE
`multiprocessing.Array` is missing

### ISSUE THREAD

**Issue by orenbenkiki** [python/typeshed#2483]
A file with a single line:

```python
from multiprocessing import Array
```
Fails to typecheck with the error:

```
example.py:1: error: Module 'multiprocessing' has no attribute 'Array'
```

This happens with both Python 3.6.3 and 3.7.0, using mypy version 0.630 which is currently the latest version.

**Comment by JelleZijlstra** [comment]
Thanks for the report! Would you mind submitting a pull request to fix it?

**Comment by orenbenkiki** [comment]
It seems like the current types only work in Python 2, and a PR would need to revamp the types for the whole module. I doubt I'll have the opportunity to do this any time soon :-(

**Comment by LouisAumaitre** [comment]
I have the same problem with `multiprocessing.Pipe`.

**Comment by markuspi** [comment]
Looks like this was fixed by #3272
